modules/services/taginfo.js

```diff
@@ -146,6 +179,23 @@ export function init() {
             });
     };
 
+    taginfo.roles = function(parameters, callback) {
```

Comment:
Can you add a test for `roles` to [`test/spec/services/taginfo.js`](https://github.com/openstreetmap/iD/blob/master/test/spec/services/taginfo.js)?